actions.js

```diff
@@ -0,0 +1,66 @@
+export function getActionDefinitions(self) {
+
+	const actions = {}
+
+	actions['send'] = {
```

Comment:
I told you already several times. This is a very generic command, turning your module in a generic tcp sender. You mustn't rely on just filling the option of this command with your presets.
Create proper actions with valid options instead.

Reply:
its ok

Reply:
it is not ok!
I tell you again and again and again and again.
You can send me pictures of your computers on slack or tell me how much your friends like your software or videos how good this is working, it will not change that this part of your code is unnecessarily generic. The user can type whatever he wants in that textinput and this is not ok.
My time is limited and I volunteer in doing this reviews, I already have spent more time in explaining your errors to you than I would have needed to write this module myself from scratch. I will not do the next review very soon. Please only request the next review if you have solved all the issues. Just because it works for you doesn't mean you have solved them.